### PR TITLE
Csp 121 course management addition

### PR DESCRIPTION
**Pull request by MarkFCain**

### CODE REVIEW

File: src/SFA.DAS.Roatp.Application/Providers/Queries/GetProvider/GetProviderQueryResult.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿namespace SFA.DAS.Roatp.Application.Providers.Queries.GetProvider
+{
+    public class GetProviderQueryResult
+    {
+        public ProviderModel Provider { get; set; }
```
Comment (DevenKShah) [inline src/SFA.DAS.Roatp.Application/Providers/Queries/GetProvider/GetProviderQueryResult.cs:5]: The model is unnecessary in my opinion. you could have just used the result object directly to have the structure you want to return.

File: src/SFA.DAS.Roatp.Application/Providers/Queries/GetProvider/ProviderModel.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
+﻿namespace SFA.DAS.Roatp.Application.Providers.Queries.GetProvider
+{
+    public class ProviderModel
+    {
+        public int Id { get; set; }
```
Comment (DevenKShah) [inline src/SFA.DAS.Roatp.Application/Providers/Queries/GetProvider/ProviderModel.cs:5]: No need to return internal ids.

**Review by DevenKShah — CHANGES_REQUESTED** [review]
Internal provider Id should not be exposed.

File: src/SFA.DAS.Roatp.Application/Providers/Queries/GetProvider/ProviderModel.cs (hunk 2 above)
Comment (MarkFCain) [reply src/SFA.DAS.Roatp.Application/Providers/Queries/GetProvider/ProviderModel.cs:5]: I will remove this

[the remark this answers is not in the document]
Comment (MarkFCain) [reply src/SFA.DAS.Roatp.Application/Providers/Queries/GetProvider/ProviderModel.cs]: removed

File: src/SFA.DAS.Roatp.Application/Providers/Queries/GetProvider/GetProviderQueryResult.cs (hunk 1 above)
Comment (MarkFCain) [reply src/SFA.DAS.Roatp.Application/Providers/Queries/GetProvider/GetProviderQueryResult.cs]: removed

File: src/SFA.DAS.Roatp.Application.UnitTests/Providers/Queries/GetProviderQueryResultTests.cs [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    [TestFixture]
+    public class GetProviderQueryResultTests
+    {
+        [Test, RecursiveMoqAutoData]
+        public void Operator_PopulatesModelFromEntity(Provider provider)
+        {
+            var model = (GetProviderQueryResult)provider;
+
+            model.Should().BeEquivalentTo(provider, c => c
+                .Excluding(s => s.Id)
```
Comment (DevenKShah) [inline src/SFA.DAS.Roatp.Application.UnitTests/Providers/Queries/GetProviderQueryResultTests.cs:18]: Alignment

**Review by DevenKShah — APPROVED** [review]

**Pull request merged**